service-discovery/README.adoc

```diff
@@ -31,304 +30,371 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
-
-Once application is running we can now connect our service using `rhoas cluster bind` command. This command will let us inject credentials to our application as files. Credentials will be by default injected as volume into our kuberentes deployment. Quarkus Kuberentes Client will detect them and automatically configure our application for us.
-
-Bind command will create following structure in your pod that many frameworks like quarkus can automatically read and autoconfigure your application to connect with the Kafka and other services.
 
+. Use the RHOAS CLI to bind your Kafka instance to your OpenShift project.
++
 [source]
 ----
-/bindings/yourkafka
-├── bootstrapServers
-├── password
-├── provider
-├── saslMechanism
-├── securityProtocol
-├── type
-└── user
+$ rhoas cluster bind
 ----
++
+You are prompted to specify the Kafka instance that you want to connect to OpenShift.
 
-.Procedure
-. Execute
+. Type the name of the Kafka instance that you want to connect to OpenShift. Press *Enter*.
+. Type `y` to confirm that you want to continue. Press *Enter*.
++
+When binding is complete, you should see output like the following:
 +
 [source]
 ----
-rhoas cluster bind
+Binding my-kafka-instance with rhoas-quarkus-kafka app succeeded
 ----
 +
-Command will automatically detect our application and single Kafka service in our namespace and inject connection details to the running application.
-. Select application we want to connect with.
-. Press enter to select it on the list
-. Verify that command finished with **Binding succeeded** message
-. Please go back to your app `prices.html` page
-. You should see prices changing on the webpage
-
-
-ifdef::qs[]
-.Verification
-* Command should end up with **Binding succeeded**
-* Can you view `prices.html` page and chaging prices
-endif::[]
-
-ifdef::qs[]
-[#conclusion]
-Congratulations! You successfully completed the {product} Service Discovery quick start, and are now ready to deploy and connect services to your own applications with {product}.
-endif::[]
+The preceding output shows that the RHOAS CLI has successfully bound a Kafka instance called `my-kafka-instance` to the example Quarkus application (called `rhoas-quarkus-kafka`) in OpenShift. As part of this process, the RHOAS Operator injects connection credentials for the Kafka instance into the Pod for the Quarkus application. Quarkus automatically detects the credentials and uses them to interact with the Kafka instance.
++
+When service binding is complete, the Quarkus application starts to use the `prices` Kafka topic that you created earlier in the tutorial. One part of the Quarkus application publishes price updates to this topic, while another part of the application consumes the updates.
 
-ifdef::parent-context[:context: {parent-context}]
-ifndef::parent-context[:!context:]
+. To verify that the Quarkus application is using the Kafka topic, reload the *Last price* web page that you opened in step 3 of this tutorial.
```

Comment:
This sounds like another step and also verification phase.

Reply:
See previous replies.